_layouts/default.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
-  {% include head.html %}
+  {% include head.html -%}
```

Comment:
Can you please explain what the hyphens after/before the open/close tags do?  I can't find it in the Jekyll docs.  Are these related to the whitespace collapse you mentioned in the PR description?

Reply:
You can find them in the liquid docs, which is the preprocessor jekyll uses: https://shopify.github.io/liquid/basics/whitespace/
They basically remove whitespace before and after the tags and were introduced in liquid 4.0.0 which is used since jekyll 3.5.0